lms/views/lti/basic_launch.py

```diff
@@ -112,24 +112,40 @@ def reconfigure_assignment_config(self):
         schema=ConfigureAssignmentSchema,
         renderer="lms:templates/lti/basic_launch/basic_launch.html.jinja2",
     )
-    def configure_assignment(self):
+    def configure_assignment_callback(self):
         """
         Save the configuration from the file-picker for future launches.
 
         We then continue as if we were already configured with this document
         and display it to the user.
         """
-        extra = {}
-        if group_set := self.request.parsed_params.get("group_set"):
-            extra["group_set_id"] = group_set
-
-        # Make any product-specific actions after configuring the assignment
-        self.request.product.plugin.misc.post_configure_assignment(self.request)
+        return self._configure_and_show_document()
 
-        return self._show_document(
-            document_url=self.request.parsed_params["document_url"],
-            assignment_extra=extra,
+    @view_config(
```

Comment:
We could re-use the existing view and have a conditional there but I reckon keeping two views it's simpler and will probably easier to refactor in the future.